src/browser-polyfill.js

```diff
@@ -30,17 +37,26 @@ if (typeof browser === "undefined" || Object.getPrototypeOf(browser) !== Object.
      * not exist when accessed, but behaves exactly as an ordinary WeakMap
      * otherwise.
      *
-     * @param {function} createItem
-     *        A function which will be called in order to create the value for any
-     *        key which does not exist, the first time it is accessed. The
-     *        function receives, as its only argument, the key being created.
+     * @template {object} K
+     * @template V
```

Comment:
This is necessary for proper IntelliSense support.

Reply:
What is "proper IntelliSense support"? What happens without these comments? We don't use IntelliSense, so such comments are not likely to be maintained in the long run. Therefore we can as well not add them in the first place.

Reply:
Without `@template`, **VS&nbsp;Code** assumes that the&nbsp;`DefaultWeakMap` is&nbsp;untyped and that the&nbsp;result of&nbsp;`get(…)` is&nbsp;`any`, since it&nbsp;can’t determine the&nbsp;values of&nbsp;`K`&nbsp;and&nbsp;`V` of&nbsp;the&nbsp;extended [`WeakMap`](https://github.com/Microsoft/TypeScript/blob/5bcf251f8ae876ae3f4fb12d8b89412d162e23ed/lib/lib.es2015.collection.d.ts#L45-L56(